yt_dlp/extractor/jwplatform.py

```diff
@@ -8,14 +8,16 @@ class JWPlatformIE(InfoExtractor):
     _VALID_URL = r'(?:https?://(?:content\.jwplatform|cdn\.jwplayer)\.com/(?:(?:feed|player|thumb|preview|manifest)s|jw6|v2/media)/|jwplatform:)(?P<id>[a-zA-Z0-9]{8})'
     _TESTS = [{
         'url': 'http://content.jwplatform.com/players/nPripu9l-ALJ3XQCI.js',
-        'md5': 'fa8899fa601eb7c83a64e9d568bdf325',
+        'md5': '3aa16e4f6860e6e78b7df5829519aed3',
         'info_dict': {
             'id': 'nPripu9l',
-            'ext': 'mov',
+            'ext': 'mp4',
```

Comment:
Double check that there's no mov now

Reply:
I wasn't sure how to check that from inside the extractor beyond verifying that running the test with ext: 'mov' fails. I removed the elespectador extractor, updated the jwplayer.py regex in _extract_embed_urls, added the elespectador URL to the set of tests, and ran them all successfully.